5eShaped/precompiled/translations/de.json

```diff
@@ -55,17 +55,17 @@
     "athletics": "Athletik",
     "deception": "Täuschung",
     "history": "Geschichte",
-    "insight": "Einsicht",
-    "intimidation": "Einschüchterung",
-    "investigation": "Investigation",
+    "insight": "Motiv erkennen",
```

Comment:
I would like to stick to single-word skills where possible.

"Einsicht" was used in 4e and is used in other german 5e sheets.